### ISSUE TITLE
Docs Bot - Slightly confusing UX auto-suggest versus llm response

### ISSUE THREAD

**Issue by richard-to** [google/mesop#848]
I noticed that when I type something, it looks like auto suggest is showing up with examples. 

I think I need to press enter to get the LLM response. 

But it could easily be confused that the auto suggest is the only response (and thus the user may not press enter / send to get the LLM response)

**Comment by wwwillchen** [comment]
yeah, it's a little confusing - any suggestion on how to make the UX clearer?
